src/services/TestNumberService.ts

```diff
@@ -39,6 +39,7 @@ export class TestNumberService {
                     .catch((error: AWSError) => {
                         console.error(error); // limit to 5 attempts
                         if (error.statusCode === 400 && error.message.toLowerCase().includes("the conditional request failed")) {
+                            console.error(`Insert attempt ${attempts + 1} of ${Configuration.getInstance().getMaxAttempts()}.`);
```

Comment:
Given this lives in the Catch statement, and there's not "insert attempt 1" message, you could simply log that you had X failures? Seems to better address the concern